src/main/java/org/spongepowered/common/command/SpongeCommandManager.java

```diff
@@ -266,6 +266,16 @@ public boolean containsMapping(CommandMapping mapping) {
 
     @Override
     public CommandResult process(CommandSource source, String commandLine) {
+        if (source instanceof Conversant) {
+            final Optional<Conversation> optionalConversation = ((Conversant) source).getConversation();
+            if (optionalConversation.isPresent()) {
+                final Conversation conversation = optionalConversation.get();
+                if (!conversation.allowsCommands()) {
+                    ((Conversant) source).sendThroughMessage(Text.of("You must exit this conversation before you can use commands!"));
```

Comment:
I'd prefer if the conversation/plugin can handle this, to allow this message to be customised.